### PR TITLE
Add story points question to agile quiz

### PR DESCRIPTION
**Pull request by gerhynes**
Checklist:

- [x] I have read [freeCodeCamp's contribution guidelines](https://contribute.freecodecamp.org).
- [x] My pull request has a [descriptive title](https://contribute.freecodecamp.org/#/how-to-open-a-pull-request?id=prepare-a-good-pr-title) (**not** a vague title like `Update index.md`)

Closes #XXXXX

### CODE REVIEW

File: src/data/agile-quiz.ts [hunk 1]
```diff
@@ -217,6 +219,20 @@ const agileQuiz = [
...
+    Answer:
+      "An estimate of the amount of effort required to deliver a user story",
+    Distractor1:
+      "An estimate of the amount of sprints required to deliver a user story",
+    Distractor2:
+      "An estimate of the amount of days required to deliver a user story",
+    Distractor3:
+      "An estimate of the amount of hours required to deliver a user story",
+    Explanation:
+      "Story points are units of measure for expressing an estimate of the overall effort required to fully implement a product backlog item or any other piece of work.",
```
Comment (jdwilkin4) [inline src/data/agile-quiz.ts:233]: Let's reword this so it is not as similar to this Atlassian article 
https://www.atlassian.com/agile/project-management/estimation#:~:text=Story%20points%20are%20units%20of,work%2C%20and%20risk%20or%20uncertainty.

```suggestion
      "Story points are used to estimate the amount of effort required to deliver a user story for a project.",
```

**Review by jdwilkin4 — CHANGES_REQUESTED** [review]
Thank you for your contribution!

Please review the requested changes.
Once those changes have been implemented, then we can merge your PR into main. 👍

**Comment by gerhynes** [comment]
Thanks for the improved wording, Jessica.

**Review by jdwilkin4 — APPROVED** [review]
Congrats on another contribution to our repository. 🎉
We look forward to future contributions. 👍

**Pull request merged**